src/redux/books/books.js

```diff
@@ -0,0 +1,45 @@
+// to add action types
+const addBook = 'add_book';
+const removeBook = 'remove_book';
```

Comment:
- [ ] Kindly make sure your const are Uppercase since global constants are always uppercase. The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like:       const ADD_BOOK = bookstore/books/ADD_BOOK and const REMOVE_BOOK = bookstore/books/REMOVE_BOOK. Kindly alter your actions to conform to the ducks pattern.